analyzer/codechecker_analyzer/analyzers/clangsa/statistics_collector.py

```diff
@@ -137,7 +137,8 @@ def checker_analyze_cfg(path):
             return []
         else:
             return ['-Xclang', '-analyzer-config',
-                    '-Xclang', 'api-metadata-path=' + path]
+                    '-Xclang',
+                    'alpha.ericsson.statisticsbased:APIMetadataPath=' + path]
```

Comment:
Shouldn't this be clang version dependent?

We are doing it in [clang_options.py](https://github.com/Ericsson/codechecker/blob/9c2ab20fa1ca818fa9c935fad3a7e832d2640bfa/analyzer/codechecker_analyzer/analyzers/clangsa/clang_options.py)

Reply:
since this checker is only available within ericsson, which will be released with clang9 anyway, this is ok.